v21.1/operate-cockroachdb-kubernetes.md

```diff
@@ -421,28 +421,93 @@ Currently, only the pods are updated with new ports. To connect to the cluster,
 ### Add nodes
 
 <section class="filter-content" markdown="1" data-scope="operator">
-Before scaling CockroachDB, ensure that your Kubernetes cluster has enough worker nodes to host the number of pods you want to add. This is to ensure that two pods are not placed on the same worker node, as recommended in our [production guidance](recommended-production-settings.html#topology).
+1. Before scaling CockroachDB, ensure that your Kubernetes cluster has enough worker nodes to host the number of pods you want to add. This is to ensure that two pods are not placed on the same worker node, as recommended in our [production guidance](recommended-production-settings.html#topology).
 
-If you want to scale from 3 CockroachDB nodes to 4, your Kubernetes cluster should therefore have at least 4 worker nodes. You can verify the size of your Kubernetes cluster by running `kubectl get nodes`. If you need to add worker nodes on GKE, for example, run this command and specify the desired number of nodes:
+    If you want to scale from 3 CockroachDB nodes to 4, your Kubernetes cluster should therefore have at least 4 worker nodes. You can verify the size of your Kubernetes cluster by running `kubectl get nodes`. If you need to add worker nodes on GKE, for example, run this command and specify the desired number of nodes:
 
-{% include_cached copy-clipboard.html %}
-~~~ shell
-gcloud container clusters resize {cluster-name} --num-nodes 4
-~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    gcloud container clusters resize {cluster-name} --region {region-name} --num-nodes 4
```

Comment:
@udnay I realized this line is now incorrect. Since we now specify a regional cluster, the `--num-nodes=4` flag will be interpreted differently. instead of scaling to 4 worker nodes across the cluster, this creates 4 worker nodes in each of the 3 zones, for a total of 12 worker nodes. is there a way to add just 1 zone to a regional cluster while keeping `--num-nodes=1`?

Reply:
I would say, change `num-nodes` to 2 and make the example 6 instead of 4 maybe. This is meant to be an example of a production ready system?

I maybe gave wrong guidance then, we could just use zonal if this is for an example and make a callout that we suggest regional GKE clusters for production use so that the k8s cluster is more resilient to zonal outages.